Packs/IdentityThreatResponsePack/Playbooks/playbook-ImpossibleTravelRiskEvent_Response_using_XSOAR_and_Silverfort.yml

```diff
@@ -0,0 +1,1099 @@
+          "x": -90,
+          "y": 920
+        }
+      }
+  "30":
+    form:
+      description: ""
+      expired: false
+      questions:
+      - defaultrows: []
+        fieldassociated: ""
+        gridcolumns: []
+        id: "0"
+        label: ""
+        labelarg:
+          simple: Get details from service account manager/owner
+        options: []
+        optionsarg: []
+        placeholder: ""
+        readonly: false
+        required: false
+        tooltip: ""
+        type: shortText
+      sender: ""
+      title: Get details from service account manager/owner
+      totalanswers: 0
+    id: "30"
+    ignoreworker: false
+    message:
+      bcc:
+      body:
+      cc:
+      format: ""
+      methods:
+      - email
+      subject:
+        simple: Get details from service account manager/owner
+      timings:
+        completeafterreplies: 1
+        retriescount: 2
+        retriesinterval: 360
+      to:
+        simple: Administrator
+    nexttasks:
+      '#none#':
+      - "9"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 8c9d315a-8b4b-4662-87f2-ae96ad2428d5
+      iscommand: false
+      name: Get details from service account manager/owner
+      type: collection
+      version: -1
+    taskid: 8c9d315a-8b4b-4662-87f2-ae96ad2428d5
+    timertriggers: []
+    type: collection
+    view: |-
+      {
+        "position": {
+          "x": -90,
+          "y": 1115
+        }
+      }
+  "35":
+    continueonerror: true
+    id: "35"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "36"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      incidentId:
+        simple: ${incident.id}
+      name: {}
+      type: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: Builtin
+      description: |-
+        A script to generate investigation summary report in an automated way
+        Can be used in post-processing flow as well.
+      id: 224d2d54-c200-474e-824b-6ccde86e380d
+      iscommand: false
+      name: Generate Investigation Summary Report
+      script: GenerateInvestigationSummaryReport
+      type: regular
+      version: -1
+    taskid: 224d2d54-c200-474e-824b-6ccde86e380d
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 162.5,
+          "y": 1855
+        }
+      }
+  "36":
+    id: "36"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "26"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      additionalHeader: {}
+      attachCIDs: {}
+      attachIDs: {}
+      attachNames: {}
+      bcc: {}
+      body: {}
+      bodyType: {}
+      cc: {}
+      flag: {}
+      from: {}
+      headers: {}
+      htmlBody: {}
+      importance: {}
+      raw_message: {}
+      replyTo: {}
+      subject: {}
+      templateParams: {}
+      to:
+        simple: pramukh@demisto.com
```

Comment:
Please make sure that you have no problem that this email address will be on a public repository.